From fe8583254b8abeb81a8767c6cd5a153f5f84abd1 Mon Sep 17 00:00:00 2001
Message-ID: <fe8583254b8abeb81a8767c6cd5a153f5f84abd1.1708289807.git.sharlatanus@gmail.com>
In-Reply-To: <b6807bf457aa3debcd60c49908ebd1b147744e20.1708289807.git.sharlatanus@gmail.com>
References: <b6807bf457aa3debcd60c49908ebd1b147744e20.1708289807.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 17 Feb 2024 00:39:17 +0000
Subject: [PATCH v2 08/10] gnu: go-github-com-songmu-gitconfig: Fix build.

As seen in <https://ci.guix.gnu.org/build/3490304/log/raw>.

* gnu/packages/golang-xyz.scm go-github-com-songmu-gitconfig: Update
style.
[arguments] <#:go>: Use go-1.21.

Change-Id: Iea7a32b51d4bd74e99c52938307c11177ae79199
---
 gnu/packages/golang-xyz.scm | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/golang-xyz.scm b/gnu/packages/golang-xyz.scm
index a6031a8354..2408916c39 100644
--- a/gnu/packages/golang-xyz.scm
+++ b/gnu/packages/golang-xyz.scm
@@ -1036,10 +1036,12 @@ (define-public go-github-com-songmu-gitconfig
         (base32 "1y01h496a7pfj1g2bclls5b0nl3vnj7nz610jj1dzq9kxrwxk7fk"))))
     (build-system go-build-system)
     (arguments
-     `(#:import-path "github.com/Songmu/gitconfig"
-       ;; Package's tests appear to be hardcoded to the author's gitconfig
-       ;; and require network access.
-       #:tests? #f))
+     (list
+      ;; Package's tests appear to be hardcoded to the author's gitconfig
+      ;; and require network access.
+      #:tests? #f
+      #:go go-1.21
+      #:import-path "github.com/Songmu/gitconfig"))
     (propagated-inputs
      (list go-github-com-goccy-yaml))
     (synopsis "Go library to get configuration values from gitconfig")
-- 
2.41.0

